packages/manager/apps/dedicated/client/app/account/otrs/otrs.controller.js

```diff
@@ -6,6 +6,8 @@ import isDate from 'lodash/isDate';
 import map from 'lodash/map';
 import set from 'lodash/set';
 
+import moment from 'moment';
```

Comment:
```suggestion

```
moment is set as a global so I'm not sure this import is needed

Reply:
Yes i confirmed it was part of window property